src/hub.js

```diff
@@ -1273,6 +1273,8 @@ document.addEventListener("DOMContentLoaded", async () => {
       if (isInitialJoin) {
         store.addEventListener("profilechanged", hubChannel.sendProfileUpdate.bind(hubChannel));
 
+        const requestedOccupants = [];
```

Comment:
is this just to avoid allocations? on the fence if we should care here given the rarity of this event, vs just doing a `.filter.map` on the `presence.state`

Reply:
presence.state is an object, so not sure if that makes this any easier to do